src/components/AddMealForm/AddMealForm.module.css

```diff
@@ -0,0 +1,13 @@
+.AddMealForm {
```

Comment:
The filename already suggests these are "AddMealForm" styles. Modules add a special prefix spontaneously, which causes that the "form" class from module A! == the "form" class from module B.

So class 'form',  'form__header' and 'form__button' would be enough IMO.